.github/workflows/node-4+.yml

```diff
@@ -2,6 +2,11 @@ name: 'Tests: node.js'
 
 on: [pull_request, push]
 
+concurrency:
+  # Pushing new changes to a branch will cancel any in-progress CI runs of this workflow
```

Comment:
i think the comments need updating

Reply:
Mmhm yup good point - do you actually want to keep it at all?

Reply:
i'm fine removing the comment :-)